lib/rules/indent.js

```diff
@@ -61,6 +66,24 @@ module.exports = function(context) {
         }
     }
 
+    if (typeof options.VariableDeclarator === "number") {
+        options.VariableDeclarator = {
+            var: options.VariableDeclarator,
+            let: options.VariableDeclarator,
+            const: options.VariableDeclarator
+        };
+    } else {
```

Comment:
No they don't because `assign` replaces the object reference (it doesn't do deep copying) so I can (and did) end up with something like `{ const: 2 }` where other values were left out.